images/macos/macos-13-arm64-Readme.md

```diff
@@ -16,22 +22,22 @@
 - GNU Fortran 11 (Homebrew GCC 11.4.0) - available by `gfortran-11` alias
 - GNU Fortran 12 (Homebrew GCC 12.3.0) - available by `gfortran-12` alias
 - GNU Fortran 13 (Homebrew GCC 13.2.0) - available by `gfortran-13` alias
-- Julia 1.9.3
-- Kotlin 1.9.10-release-459
+- Julia 1.9.4
+- Kotlin 1.9.255-SNAPSHOT
```

Comment:
https://youtrack.jetbrains.com/issue/KT-63139/Incorrect-kotlin-implementation-version-1.9.255-SNAPSHOT-in-metadata-info